examtool/examtool/api/convert.py

```diff
@@ -445,7 +447,7 @@ def _convert(text, *, path=None):
     }
 
 
-def pandoc(target, *, draft=False):
+def pandoc(target, *, draft=False, threadcount=16):
```

Comment:
nit: `thread_count` or `num_threads`?

Reply:
Ya that is the phrase I was looking for.